### DISCUSSION TITLE
华为pura70 pro进度集显示异常

### DISCUSSION THREAD

**Discussion by shaneshuo** [WatsonYao/nian_app#116] [Q&A] [answered]
作者您好，我手机型号是华为pura70pro，进度集的数字在点完打卡后消失，也没有颜色，只有9 19 29是以前显示的样子。尝试安装了有进度集的第一个版本，也是这样。之前的荣耀手机没有类似问题
![Screenshot_20241126_202443_sa nian so](https://github.com/user-attachments/assets/e39a34c8-3263-4c4a-8adc-458ae442556b)

**Comment by WatsonYao** [comment, marked as answer]
暂时查不到原因，不清楚系统这块是不是有什么优化之类的。
